engine/src/main/java/org/pentaho/di/trans/TransMeta.java

```diff
@@ -2783,6 +2784,10 @@ public TransMeta( String fname, IMetaStore metaStore, Repository rep, boolean se
     // OK, try to load using the VFS stuff...
     Document doc = null;
     try {
+      if (parentVariableSpace == null ) {
```

Comment:
Would we need this for jobs as well?

Reply:
apparently it as the variable space when running the job